PhpAmqpLib/Wire/AMQPAbstractCollection.php

```diff
@@ -193,6 +193,28 @@ public function __construct(array $data = null)
         return $val;
     }
 
+    public function offsetExists($offset)
+    {
+        return isset($this->data[$offset]);
+    }
+
+    public function offsetGet($offset)
+    {
+        $value = $this->data[$offset];
```

Comment:
What happens if that `$offset` is not defined? Would be better not to throw undefined index notice.

Reply:
fixed